### PR TITLE
Change address validation

### PR DESCRIPTION
**Pull request by tmccoy529**
Added validation to determine if address is a valid address inside of GIS database. This triggers when there is no response from the endpoint. If there is a response then we can assume the address is valid and then move on. This handles anyone silly enough to side step the auto complete and manually type in their address.

### CODE REVIEW

File: src/components/forms/BasicInformationForm.jsx [hunk 1]
```diff
@@ -19,17 +20,24 @@ const BasicInformationForm = props => {
...
       }}
       validationSchema={Yup.object({
         businessName: Yup.string()
           .transform(value => (!value ? null : value))
           .required("Required"),
-        businessAddress: Yup.string()
-          .transform(value => (!value ? null : value))
+        businessAddress: Yup.mixed()
           .required("Required")
+          .test("is-valid-address", "Please enter a valid address.", value => {
```
Comment (martypowell) [inline src/components/forms/BasicInformationForm.jsx]: param here should change from `value` to `address` or `addressValue`

Comment (martypowell) [reply src/components/forms/BasicInformationForm.jsx]: So when you are validating the address, it takes a little time, but there is no visual indication taht is something is happening, we should probably add a checking message. Maybe soemthing like "Validating address..."

Comment (tmccoy529) [reply src/components/forms/BasicInformationForm.jsx]: added

**Comment by martypowell** [comment]
@baltimorecounty/front-end-developers this is ready for another look.

File: src/components/forms/BasicInformationForm.jsx [hunk 2]
```diff
@@ -52,6 +83,12 @@ const BasicInformationForm = props => {
               name="businessAddress"
               label="Business Address"
             />
+            {isValidAddressMessage && (
+              <p role="alert" className="error-message">
```
Comment (martypowell) [inline src/components/forms/BasicInformationForm.jsx]: This probably should go into the error message comopnet someone, i can take a look at this real quick.

File: src/components/forms/BasicInformationForm.jsx [hunk 3]
```diff
@@ -17,19 +18,50 @@ const BasicInformationForm = props => {
...
+
   return (
     <Formik
-      initialValues={{ businessName: "", businessAddress: "" }}
-      onSubmit={values => {
-        onValidSubmission(values);
+      initialValues={{
+        businessName: "",
+        businessAddress: "",
+        businessAddressParts: {}
```
Comment (sgrg1) [inline src/components/forms/BasicInformationForm.jsx]: Do we really need businessAddressParts:{} in initialValues? I don't see the use of it here.

Comment (martypowell) [reply src/components/forms/BasicInformationForm.jsx]: i can't remember why, but i know ive personally run into an issue with teh form values we are trying to capture is not specified. At a minimum, if we do include initial values, you know what fields the form is setting without having to inspect the values.

**Pull request merged**